Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -149,26 +153,34 @@ def keg_contain?(string, keg, ignores, formula_and_runtime_deps_names = nil, arg
         until io.eof?
           str = io.readline.chomp
           next if ignores.any? { |i| i =~ str }
-          next unless str.include? string
+          next unless str.match? Keg.relocatable_path_regex(string)
```

Comment:
Is it work caching `Keg.relocatable_path_regex(string)` given it's used multiple times?

Reply:
Probably not necessary but can't hurt I guess